app/js/util/constants.js

```diff
@@ -0,0 +1,9 @@
+export const CONSERVATIVE = Symbol.for('Conservatives');
+export const NDP = Symbol.for('NDP');
+export const LIBERAL = Symbol.for('Liberals');
```

Comment:
Confusingly, no.

```js
Symbol('abc') !== Symbol('abc')
// but
Symbol.for('abc') === Symbol.for('abc')
```

Essentially, using `.for` stores it in the global symbol scope, but creating a symbol without `.for` installs it locally only, and it can never be recreated for comparison…it'll be forever unique.